### PR TITLE
feat: allow users to add arialabel to text input

### PR DESCRIPTION
**Pull request by akoreman**
*Issue #, if available:*

*Description of changes:* Currently, the aria-label of the text input says `Cursor at row x`, some users might want to add context about their editor to this aria label. This change adds an option, `textInputAriaLabel`, which is used to create and aria-label as `textInputAriaLabel, Cursor at row x`.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

Pull Request Checklist:
* [ ] No backwards incompatible changes were made to Ace's public interface which is defined through the main typings file (`ace.d.ts`) and its references:
    * https://github.com/ajaxorg/ace/blob/master/ace.d.ts 
    * https://github.com/ajaxorg/ace/blob/master/ace-modes.d.ts 
    * https://github.com/ajaxorg/ace/blob/master/ace-extensions.d.ts

### CODE REVIEW

File: ace.d.ts [hunk 1]
```diff
@@ -235,6 +235,7 @@ export namespace Ace {
     relativeLineNumbers: boolean;
     enableMultiselect: boolean;
     enableKeyboardAccessibility: boolean;
+    textInputAriaLabel: string;
```
Comment (marinsokol5) [inline ace.d.ts:238]: So I guess nothing is optional here? Although everything is

File: src/keyboard/textinput.js [hunk 2]
```diff
@@ -90,7 +90,8 @@ TextInput= function(parentNode, host) {
             text.setAttribute("aria-roledescription", nls("text-input.aria-roledescription", "editor"));
             if(host.session) {
                 var row =  host.session.selection.cursor.row;
-                text.setAttribute("aria-label", nls("text-input.aria-label", "Cursor at row $0", [row + 1]));
+                var arialLabel = `${host.$textInputAriaLabel}, ${nls("text-input.aria-label", "Cursor at row $0", [row + 1])}`;
```
Comment (marinsokol5) [inline src/keyboard/textinput.js:93]: should we hide "," if there is textInputAriaLabel is not there? Basically to cater for the initial value.

File: src/keyboard/textinput_test.js [hunk 3]
```diff
@@ -753,6 +753,18 @@ module.exports = {
         assert.equal(editor.getValue(), "");
         sendEvent("input", {key: {inputType: "historyRedo"}});
         assert.equal(editor.getValue(), "x");
+    },
+
+    "test: text input aria label": function() {
```
Comment (marinsokol5) [inline src/keyboard/textinput_test.js:758]: Can we also add same test but without `textInputAriaLabel ` being set.

Comment (akoreman) [reply src/keyboard/textinput_test.js]: good point, added

File: src/keyboard/textinput.js (hunk 2 above)
Comment (akoreman) [reply src/keyboard/textinput.js]: yeah that's fair, changed

File: ace.d.ts (hunk 1 above)
Comment (marinsokol5) [reply ace.d.ts:238]: I see we have it in https://github.com/ajaxorg/ace/blob/master/ace.d.ts#L1104

**Review by marinsokol5 — APPROVED** [review]

**Pull request merged**